src/Database/Migrator.php

```diff
@@ -63,6 +63,9 @@ public function __construct(
         $this->repository = $repository;
 
         $this->schemaBuilder = $connection->getSchemaBuilder();
+
+        // Workaround for https://github.com/laravel/framework/issues/1186
+        $connection->getDoctrineSchemaManager()->getDatabasePlatform()->registerDoctrineTypeMapping('enum', 'string');
```

Comment:
As per [this comment](https://github.com/laravel/framework/issues/1186#issuecomment-379640952), this should not be needed anymore?

Reply:
It wasn't working when I tried it, but we should probably double-check

Reply:
Just tested with DBAL 2.8.0; still not working.